public/js/user/requestDom.js

```diff
@@ -0,0 +1,55 @@
+// const personalDetails = document.querySelector('.personal--details');
+const next = document.querySelector('.next--button');
+const {
+  pesonalSection, requestSection,
+  personalDetails, yourRequest, fullNameSpan, phoneNumberSpan, cityNameSpan, streetNameSpan, requestSubmit,
+} = querySelectors(['pesonalSection', 'requestSection', 'personalDetails', 'yourRequest', 'fullNameSpan', 'phoneNumberSpan', 'cityNameSpan', 'streetNameSpan', 'requestSubmit'], ['.pesonal--section', '.request--section', '.personal--details', '.your--request', '.full--name-span', '.phone--number-span', '.city--name-span', '.street--name-span', '.request--submit']);
+
+next.addEventListener('click', (e) => {
+  e.preventDefault();
```

Comment:
@Jamalat-shamallakh At the top of this function, it might be a good idea to "clear out" any previous error text:
```
fullNameSpan.textContent = ''
phoneNumberSpan.textContent = ''
...
...
```

So that "old" errors do not persist.  That would avoid a bug where a person tries to submit and has two errors (like no name and an incorrect phone number), they fix one error (they enter a name) and click to submit again, and the name error text is still showing.